test/shared/src/main/scala/zio/test/ZIOSpecDefault.scala

```diff
@@ -7,7 +7,7 @@ import zio.stacktracer.TracingImplicits.disableAutoTrace
 
 abstract class ZIOSpecDefault extends ZIOSpec[TestEnvironment] {
 
-  override val bootstrap: ZLayer[Scope, Any, TestEnvironment] =
+  override val bootstrap: ZLayer[Any, Any, TestEnvironment] =
```

Comment:
Isn't this technically a breaking change?

Reply:
It is binary compatible. It is potentially source incompatible if you are requiring a `Scope` as an input to your layer, but I think this is an anti-pattern because scopes should be embedded in the scope of the layer using `ZLayer.scoped` instead of appearing as inputs to layers.